bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TT_SStt/TT_SStt_customizecards.dat

```diff
@@ -0,0 +1,10 @@
+set param_card decay 6000006 0.1
+set param_card decay 6100001 0.01
```

Comment:
will you use the cross section from sample production in your analysis, if yes, are those width chosen randomly small